wallet/src/de/schildbach/wallet/data/DashPayProfile.kt

```diff
@@ -16,18 +16,28 @@ data class DashPayProfile(@PrimaryKey val userId: String,
                           val createdAt: Long = 0,
                           val updatedAt: Long = 0) : Parcelable {
     companion object {
-        fun fromDocument(document: Document, username: String): DashPayProfile? {
-            return try {
-                DashPayProfile(document.ownerId,
-                        username,
-                        document.data["displayName"] as String,
-                        document.data["publicMessage"] as String,
-                        document.data["avatarUrl"] as String,
-                        if (document.createdAt != null) document.createdAt!! else 0L,
-                        if (document.updatedAt != null) document.updatedAt!! else 0L)
-            } catch (e: Exception) {
-                null
+
+        private fun getField(document: Document, field: String, defaultValue: String = ""): String {
```

Comment:
The `defaultValue` argument is not being used.

Reply:
oh wow, good catch.

Reply:
https://github.com/dashevo/dash-wallet/pull/513/commits/e9945008258db372d2274ab3ef48ee5a191b6b8f